src/DragNDrop/ComponentTaskNode.tsx

```diff
@@ -23,16 +23,18 @@ import {
   RefreshCcw,
   EyeIcon,
   CircleDashed,
+  SettingsIcon,
 } from "lucide-react";
 
 import ArgumentsEditorDialog from "./ArgumentsEditorDialog";
-
+import { Button } from "@/components/ui/button";
+import ExecutionDetailsSheet from "@/components/ExecutionDetailsSheet";
 const inputHandlePosition = Position.Top;
 const outputHandlePosition = Position.Bottom;
 
 type InputOrOutputSpec = InputSpec | OutputSpec;
 
-const NODE_WIDTH_IN_PX = 180;
+const NODE_WIDTH_IN_PX = 200;
```

Comment:
JFYI: Be careful with coordinates and sizes. There is "snap to grid" feature and also the coordinates and sizes are saved in the pipeline. Just keep that in mind.